### PR TITLE
Setup linting in the project

### PR DESCRIPTION
**Pull request by lacasera**
#### Checklist:

- [ ] All test are passing. 
- [ ] Tested changes locally.
- [ ] Current changes in the pr have the necessary test.
- [ ] Api Documentation follows api standards.
- [ ] Improved documentation or readme where necessary. 
- [ ] Your commit messages format follows our commit message guideline.

#### What does this pr do?
adds linting to project

#### How should this be manually tested?
run `npm install`
1. edit `src/app.js` by adding some spaces
2. run `npm run  lint`
3. notice the file is linted and the spaces you added removed

#### What are the relevant issues?
#6

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const express = require('express')
-const app = express();
+const app = express()
```
Comment (kenmartey) [inline src/app.js:2]: I guess according to the linting rules, there is supposed to be a ; after each expression

File: server.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-const app = require('./app')
+const app = require('./src/app')
 
-app.listen('8080', function () {
+app.listen('8081', function () {
     console.log('App is listening on port 8080')
```
Comment (kenmartey) [inline server.js:4]: The port has been changed to listen on 8081, and the console.log still shows a message of listening to port 8080.

File: src/app.js (hunk 1 above)
Comment (lacasera) [reply src/app.js:2]: https://standardjs.com/rules.html

File: server.js (hunk 2 above)
Comment (lacasera) [reply server.js:4]: the port will be eventually moved to an .env key. so its not really an issue

Comment (kenmartey) [reply server.js:4]: Ok great.

**Pull request merged**